src/components/shared/display-range-date.tsx

```diff
@@ -0,0 +1,31 @@
+import PropTypes from "prop-types"
+import { convertToFullDate, shortenFormatDate } from "../../utils/format-date"
+
+interface DateRangeDisplayProps {
+  label: string
+  startDate?: string
+  endDate?: string
+  isMobile: boolean
+}
+
+export const DateRangeDisplay = ({
+  label,
+  startDate,
+  endDate,
+  isMobile,
+}: DateRangeDisplayProps) => (
+  <div className='flex gap-3'>
+    <div className='font-bold'>
+      {!isMobile ? <span>{label}</span> : <span>{label.trim().split(" ")[1]}</span>}:
+    </div>
+    {isMobile ? shortenFormatDate(startDate) : convertToFullDate(startDate)} to{" "}
+    {isMobile ? shortenFormatDate(endDate) : convertToFullDate(endDate)}
+  </div>
+)
+
+DateRangeDisplay.propTypes = {
```

Comment:
do we need this?

Reply:
I created this for use across multiple pages, as I have two pages that employ it due to the distinct date formats on desktop and mobile devices.

Reply:
I mean, we already have the interface `DateRangeDisplayProps` 🤔

Reply:
Ahhhh okey2 getss xD